### PR TITLE
Try travis as job

### PR DESCRIPTION
**Pull request by harikt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
-  - phpunit --coverage-clover=coverage.clover
+  - ./vendor/bin/phpunit --coverage-clover=coverage.clover
 after_script:
   - wget https://scrutinizer-ci.com/ocular.phar
   - php ocular.phar code-coverage:upload --format=php-clover coverage.clover
+
+jobs:
+  include:
+    - stage: travis
+      dist: precise
```
Comment (harikt) [inline .travis.yml]: Hi @jakejohns , 

I was looking at your issue on https://github.com/auraphp/Aura.Html/pull/54#issuecomment-316886798 , thought I could experiment a bit on it.

But looks like I failed for the current time. I will do some more experiments to see if there is a way to use multiple distro on a single travis file.

**Comment by jakejohns** [comment]
hey did you just figure it out?! It looks like maybe that worked!

**Comment by harikt** [comment]
Hey  @jakejohns , 

Yes :) .

Trying one more... https://github.com/auraphp/Aura.Html/pull/56

**Comment by harikt** [comment]
New PR will follow.

**Pull request closed without merging**